tests/installation/installation_overview.pm

```diff
@@ -24,7 +24,10 @@ sub run {
     record_soft_failure('bsc#1054974') if get_var('ALL_MODULES');
     # overview-generation
     # this is almost impossible to check for real
-    assert_screen "installation-settings-overview-loaded";
+    # During upgrade, it takes longer time to check package dependencies,
+    # especially with multiple addons selected
+    my $timeout = get_var('UPGRADE') ? 60 + 30 * (scalar split(/,/, get_var('SCC_ADDONS', get_var('ADDONS', get_var('ADDONURL', ''))))) : undef;
```

Comment:
I'm not sure if we need such a complicated structure here, it should be fine to increase timeout to some acceptable max value.

Reply:
Yes, I just didn't want to change the default timeout of installation.
Simply set timeout to 150s, since 130s is required at worst case observed.